dotnet/src/VectorDataIntegrationTests/VectorDataIntegrationTests/VectorSearch/VectorSearchDistanceFunctionComplianceTests.cs

```diff
@@ -175,7 +175,7 @@ private VectorStoreRecordDefinition GetRecordDefinition(string distanceFunction)
             ]
         };
 
-    public class SearchRecord
+    private sealed class SearchRecord
```

Comment:
Any reason to make this private? This test class (VectorSearchDistanceFunctionComplianceTests) is a base class intended to be extended by specific connectors, so it makes sense they might want to e.g. add a test of their own which references SearchRecord...